platform/android/export/export_plugin.cpp

```diff
@@ -3903,13 +3927,16 @@ Error EditorExportPlatformAndroid::export_project_helper(const Ref<EditorExportP
 		} else {
 			print_verbose(build_project_output);
 		}
+#endif
```

Comment:
nit: this block can be moved below where we run the `copy` gradle command to keep the symmetry with the way we run the Android gradle builds.